index.html

```diff
@@ -1,417 +1,245 @@
+                  <p class="post-title">Profesional Art Printing Data</p>
+                  <p class="post-description description-text">A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                  <ul class="post-tags ul-list horizontal-list">
+                    <li class="list-item-text">Html</li>
+                    <li class="list-item-text">Bootstrap</li>
+                    <li class="list-item-text">Ruby</li>
+                  </ul>
+                </div>
+                <button type="button" name="button" class="enable-button post-button">See project</button>
+              </article>
+              <article class="post-article grid-item">
+                <figure class="post-image post-image-desktop">
+                  <img src="images/img-placeholder-articles-03.png" alt="Placeholder image 03">
+                </figure>
+                <div class="post-information">
+                  <p class="post-title">Profesional Art Printing Data</p>
+                  <p class="post-description description-text">A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                  <ul class="post-tags ul-list horizontal-list">
+                    <li class="list-item-text">Html</li>
+                    <li class="list-item-text">Bootstrap</li>
+                    <li class="list-item-text">Ruby</li>
+                  </ul>
+                </div>
+                <button type="button" name="button" class="enable-button post-button">See project</button>
+              </article>
+              <article class="post-article grid-item">
+                <figure class="post-image post-image-desktop">
+                  <img src="images/img-placeholder-articles-04.png" alt="Placeholder image 04">
+                </figure>
+                <div class="post-information">
+                  <p class="post-title">Profesional Art Printing Data</p>
+                  <p class="post-description description-text">A daily selection of privately personalized reads; no accounts
+                    or sign-ups required. has been the industry's standard</p>
+                  <ul class="post-tags ul-list horizontal-list">
+                    <li class="list-item-text">Html</li>
+                    <li class="list-item-text">Bootstrap</li>
+                    <li class="list-item-text">Ruby</li>
+                  </ul>
+                </div>
+                <button type="button" name="button" class="enable-button post-button">See project</button>
+              </article>
+              <article class="post-article grid-item">
+                <figure class="post-image post-image-desktop">
+                  <img src="images/img-placeholder-articles-05.png" alt="Placeholder image 05">
+                </figure>
```

Comment:
The placeholder image 05 is currently with a wrong path, which is causing a bug in the card display. Please fix this to keep your work cads nicely aligned. 😀 

![image](https://user-images.githubusercontent.com/42799579/193386732-01e4934f-07da-4ebc-b5f8-6c6faa72bbd2.png)